cmd/resolve_bitcoin_address.go

```diff
@@ -0,0 +1,75 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/keep-network/keep-ecdsa/config"
+	"github.com/keep-network/keep-ecdsa/pkg/chain/bitcoin"
+	"github.com/keep-network/keep-ecdsa/pkg/extensions/tbtc/recovery"
+
+	"github.com/urfave/cli"
+)
+
+var ResolveBitcoinBeneficiaryAddressCommand cli.Command
+
+const resolveBitcoinAddressDescription = `Uses details provided in the configuration ` +
+	`file to estimate next Bitcoin beneficiary address that will be used for liquidation ` +
+	`recovery.`
```

Comment:
Do you think it's worth being specific here? We could mention the exact configuration path `Extensions.TBTC.Bitcoin.BeneficiaryAddress`